src/onetasker/tests.py

```diff
@@ -81,51 +89,100 @@ def test_onetasker_dashboard(self):
     def test_onetasker_tasks(self):
         onetasker_tasks = core_logic.onetasker_tasks(self.user)
         active_tasks = onetasker_tasks.get('active')
+
         for task in active_tasks:
             # decision page
-            response = self.client.get(reverse('onetasker_task_hub', kwargs={'assignment_type': task.get('type'), 'assignment_id': task.get('assignment').id}))
+            response = self.client.get(reverse('onetasker_task_hub', kwargs={
+                'assignment_type': task.get('type'),
+                'assignment_id': task.get('assignment').id}))
             content = response.content
             self.assertEqual(response.status_code, 200)
             self.assertEqual("403" in content, False)
-            self.assertEqual("You can accept or reject this task" in content, True)
+            self.assertTrue("You can accept or reject this task" in content)
             self.assertEqual("I Accept" in content, True)
```

Comment:
What you did was great, however it has as much effect as assertEqual(something in something, True). I don't think you should go around changing these, as there is probably to many of them, but it would be nice if we used assert statements that appropriate, in this case, assertTrue(something, in something).